src/Bridge/Doctrine/EventListener/PublishMercureUpdatesListener.php

```diff
@@ -145,7 +145,10 @@ private function storeEntityToPublish($entity, string $property): void
         }
 
         if ('deletedEntities' === $property) {
-            $this->deletedEntities[$this->iriConverter->getIriFromItem($entity, UrlGeneratorInterface::ABS_URL)] = $value;
+            $this->deletedEntities[(object) [
```

Comment:
can we transform this to a real class? We had issues in the past with StdClass and it's better for debugging.

Reply:
Does it worth it? It's purely internal state.

Reply:
It was in the serialization context, and indeed an anonymous class. `\stdClass` is serializable: https://3v4l.org/2VWST